packages/browser/src/core/user/index.ts

```diff
@@ -128,22 +141,26 @@ export class Cookie extends Store {
   remove(key: string): void {
     return jar.remove(key, this.opts())
   }
-}
 
-class NullStorage extends Store {
-  get = (_key: string): null => null
-  set = (_key: string, _val: unknown): null => null
-  remove = (_key: string): void => {}
+  getType = (): StoreType => 'cookie'
 }
 
 export class LocalStorage extends Store {
```

Comment:
Maybe we can remove some of the WTFs from this class:
```ts
  get<T>(key: string): T | null {
    const val = localStorage.getItem(key) // this should be wrapped in a try/catch in case storage is not available
    if (val) {
      try {
        return JSON.parse(val)
      } catch (e) {
          return JSON.parse(JSON.stringify(val)) // this is a serious code smell =( I have no idea what this is.
         }
    }
    return null
  }
```

Reply:
We can probably refactor in a subsequent PR. This code has changed since the comment, and don't want to complicate this PR further.